docs/interfaces/desktopagent.md

```diff
@@ -31,10 +31,28 @@ A Desktop Agent can be connected to one or more App Directories and will use dir
 
 ▸ **broadcast**(context: *[Context](../#context)*): `void`
 
-*Defined in [interface.ts:98](/src/interface.ts#L98)*
+*Defined in [interface.ts:162](/src/interface.ts#L162)*
 
 Publishes context to other apps on the desktop.
 
+```javascript
+agent.broadcast({version:'1.0.0',
+     entities:[
```

Comment:
I think we need to be really careful with context examples, until we have a clearer idea of what contexts look like, and also consistency with the context data working group's current draft specification.

I would be tempted to just "indicate" a context object in the examples without elaborating what is inside it, it isn't important for illustrating API examples anyway, it could just be "an object" for now.